SocialFilter/sections/general_url.txt

```diff
@@ -1,6 +1,7 @@
 !
 ! Contains unsorted urls
 !
+||share.my-plugin.com/share.min.js
```

Comment:
It seems that this rule breaks a website.
For example, if you click on a newspaper, it should show popup with clicked fragment, but it doesn't work.

Reply:
Something like this should work: `ejjdin.com##.my-plugin-share`